classes/controller/AdminController.php

```diff
@@ -1108,7 +1108,15 @@ public function processDelete()
                 }
                 $this->errors[] = $this->trans('An error occurred during deletion.', array(), 'Admin.Notifications.Error');
                 if ($res) {
-                    PrestaShopLogger::addLog(sprintf($this->l('%s deletion', 'AdminTab', false, false), $this->className), 1, null, $this->className, (int) $this->object->id, true, (int) $this->context->employee->id);
+                    PrestaShopLogger::addLog(
+                        $this->trans('%s deletion', [$this->className]),
```

Comment:
Seems like you lost the "domain" in the process `AdminTab` ?

Reply:
That is not a domain but a "context" which only makes sense when using old-styled translation files which, incidentally, no longer exist for core translations in 1.7. 

In practice, the former call falls back to a `trans()` call with no domain. Therefore, the absence of domain in this change would be correct. In fact this wording is already present in `messages.xlf`.

Fun fact: when mapping calls to `$this->l()` in a module to the new system, I use that value as a subdomain, in this case it would be `Module.Mymodule.Admintab`. Magic!

Reply:
haha, a bit cheating But alright :)